sdk/storage/Azure.ResourceManager.Storage/CHANGELOG.md

```diff
@@ -10,6 +10,18 @@
 
 ### Other Changes
 
+## 1.0.1 (2022-09-23)
```

Comment:
This is the change log of `Azure.ResourceManager.Storage`. Are you sure you want to update this file?

Reply:
This is the wrong file i updated, reverting and updating the right changelog file